src/main/java/uk/gov/gchq/palisade/service/request/DataRequestResponse.java

```diff
@@ -49,14 +50,6 @@ public DataRequestResponse token(final String token) {
         return this;
```

Comment:
Use setters

Reply:
line 48 needs a setter and shouldnt check nulls

Reply:
needs the generated annotation